lib/discourse_webauthn.rb

```diff
@@ -68,7 +72,18 @@ def self.challenge(user, secure_session)
   end
 
   def self.rp_id
-    Discourse.current_hostname
+    Rails.env.production? ? Discourse.current_hostname : "localhost"
+  end
+
+  def self.origin
+    case Rails.env
+    when "development"
+      "http://localhost:4200"
+    when "test"
+      "http://localhost:3000"
```

Comment:
I think the host and the ports can all be configured locally so this might need to reference some variables instead of hardcoding the defaults.

Reply:
Are you suggesting using ENV variables here or just lifting the values up into variables higher up this file? It does make sense but we'd need three variables, in theory: host (localhost), port and protocol.

Reply:
Just to confirm, is this origin used to check against the origin which the passkey was generated for? My initial concern is that port 4200 is the default ember cli port but what if someone uses the `-p 5000` opts. Will this break stuff?

Reply:
Yes, the port is included in the generated key so a key generated under port 4200 will raise an "origin mismatch" error if server is running on a different port. 

I'll add a note here for now. Not against improving this in a future PR.